rasa/nlu/utils/bilou_utils.py

```diff
@@ -0,0 +1,149 @@
+from typing import List, Tuple, Text, Optional, Dict, Set
+
+from rasa.nlu.tokenizers.tokenizer import Token
+from rasa.nlu.training_data import Message
+from rasa.nlu.training_data import TrainingData
+from rasa.nlu.constants import ENTITIES, TOKENS_NAMES, TEXT, BILOU_ENTITIES
+
+BILOU_PREFIXES = ["B-", "I-", "U-", "L-"]
+
+
+def bilou_prefix_from_tag(tag: Text) -> Optional[Text]:
+    """Remove the BILOU prefix from the given tag."""
```

Comment:
why? it is very clear what tag is. It is not a public API method. Then we need to add google docstrings everywhere, otherwise it doesn't make sense to add here